support/test-iso/user.nix

```diff
@@ -0,0 +1,21 @@
+{ config, pkgs, ... }:
+
+{
+  users.users = {
+    "${config.jovian.steam.user}" = {
+      isNormalUser = true;
+      uid = 1000;
+      extraGroups = [
+        "audio"
+        "networkmanager"
+        "video"
+        "wheel"
+      ];
+      initialHashedPassword = "";
```

Comment:
I'd add some default password and also enable openssh, see https://github.com/Jovian-Experiments/Jovian-NixOS/pull/141#issuecomment-1681156533

Reply:
hmmm... I generally dislike default passwords, but given `Jovian-NixOS-FOR-TESTING-ONLY` I guess it would be fine.

Though for logs, I used `sudo systemctl stop greetd` to get it to stop taking over.

Reply:
> Though for logs, I used `sudo systemctl stop greetd` to get it to stop taking over.

It's really difficult to do when greetd restarts every second...

Reply:
oh right, I assumed it was still the message with the timer.